### PR TITLE
S03 us27 03 formulary security stages

### PR DESCRIPTION
**Pull request by NEL45**

### CODE REVIEW

**Review by bouboumael — DISMISSED** [review]
ok just course.sql is here because not fix for create this formulary

File: src/Controller/StageFormController.php [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+        if (empty($stage['phone'])) {
+            $errors[] = 'Un numéro de téléphone est requis';
+        }
+
+        if (!filter_var($stage['email'], FILTER_VALIDATE_EMAIL)) {
+            $errors[] = 'Votre adresse email n\'est pas valide';
+        }
+
+        return $errors;
+    }
```
Comment (LucyMerlier) [inline src/Controller/StageFormController.php:108]: "isStillEmpty"? + if the goal is to check for empty inputs, why is there a mail validation error handling in there?
+ if i remember well, don't you have a form validation service somewhere in your project that does all this already?

File: src/View/User/Includes/_inscription_form_course.html.twig [hunk 2]
```diff
@@ -57,12 +57,10 @@
                 </div>
                 <div class="mb-3 field">
                     <label for="parentfirstname" class="form-label">
-                        <span class="text-danger">*</span>Prénom :</label>
                     <input type="text" class="form-control" id="parentfirstname" name="parentfirstname" placeholder="Laurie" required value="{{ course.parentfirstname }}" aria-required="true">
                 </div>
                 <div class="mb-3 field">
                     <label for="parentlastname" class="form-label">
-                        <span class="text-danger">*</span>Nom :</label>
```
Comment (LucyMerlier) [inline src/View/User/Includes/_inscription_form_course.html.twig:75]: wait, so there are no label content and closing tags anymore?

**Review by LucyMerlier — CHANGES_REQUESTED** [review]
see comments

File: src/Controller/StageFormController.php (hunk 1 above)
Comment (NEL45) [reply src/Controller/StageFormController.php:108]: No, we haven't got, now it's in an controller.

**Review by bouboumael — APPROVED** [review]

**Review by LucyMerlier — APPROVED** [review]

**Pull request merged**